pages/_app.js

```diff
@@ -17,6 +17,8 @@ export default function App({ Component, pageProps }) {
   const flashCardsToShow =
     router.pathname === "/archive" ? archivedFlashCards : activeFlashCards;
 
+  const noCards = flashCards.length === 0 ? true : false;
```

Comment:
There is no need for `? true : false` since `flashCards.length === 0` already returns a boolean.

```suggestion
  const noCards = flashCards.length === 0
```